docs/content/mixin-commands.md

```diff
@@ -159,43 +169,43 @@ upgrade:
 - helm:
     description: "Upgrade MySQL"
     name: porter-ci-mysql
+    replace: true
+    set:
+      mysqlPassword: "updatedtopsecret"
```

Comment:
Maybe in a quick follow-up we should make this a parameter instead of hard-coded so that people don't get confused about why we are bothering to output it?

Reply:
Also would we get any outputs from upgrade if we don't have both a step level outputs that emits `mysql-root-password` and `mysql-password` like install does, plus a a global outputs section with applyTo which includes upgrade?

Reply:
I can add a params section to the example snippet, certainly.

As this doc appeared to just focus on mixin-level outputs (and the corresponding mixin output files), I'm omitting mention of global outputs section here.  Indeed, they would need to be declared in order to be retrieved by `porter bundle outputs list`.